problem_builder/public/js/step.js

```diff
@@ -110,20 +105,15 @@ function MentoringStepBlock(runtime, element) {
             return $('.sb-step', element).data('has-question');
         },
 
-        updateChildren: function() {
-            children.forEach(function(child) {
-                var type = $(child.element).data('block-type');
-                switch (type) {
-                case 'video':
-                    updateVideo(child);
-                    break;
-                case 'sb-plot':
-                    updatePlot(child);
-                    break;
-                }
-            });
-        }
+        show: function () {
+            $(element).show();
+            childManager.show();
```

Comment:
There is a ``MentoringWithStepsBlock.hideAllSteps`` function so plain ``hideAll`` would be misleading, I decided to rename this one to ``hideStep`` (other option --- like: ``hideAllChildren`` or ``hideStepAndChildren`` --- would be too verbose.)